static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -285,10 +285,14 @@ const StyledSidebarItem = styled(Link)`
   }
 
   &:hover,
-  &:focus {
+  &.focus-visible {
     color: ${p => p.theme.white};
   }
 
+  &:focus {
+    outline: none;
+  }
+
```

Comment:
to fix this
<img width="217" alt="image" src="https://user-images.githubusercontent.com/44172267/233206647-113b71f2-1b2a-4ec1-83bf-da55b836729f.png">